masters-golf.md

```diff
@@ -0,0 +1,105 @@
+| Content | Description |
+|---|---|
+| Title               | Swinging into the Future - Golf and Blockchain Technology |
+| Author              | TallAmericano |
+| Featured text       | Is it possible for Golf to be revolutionized by blockchain technology |
+| CT link             | FIXME |
+| Publish date        | 2024-04-21 |
+
+| Social Media | Links |
+|---|---|
+| YouTube link        | FIXME |
+| Podcast link        | FIXME |
+| Bison Relay post    | Title |
+| Reddit link         | FIXME |
+| CT tweet            | FIXME |
+| YouTube tweet       | FIXME |
+| Twitter full video  | FIXME |
+| Twitter clip 1      | FIXME |
+| Twitter clip 2      | FIXME |
+| Matrix post         | Promote your main tweet in "DCR Social Media" channel |
+
+
+## Thesis statement
+
+This article explores the potential integration of blockchain technology into the Golf world
+
+
+## Pre-discussion, research, and questions
+
+* What questions are you aiming to find the answers to?
+  Could the game of golf benefit from blockchain technologies
+  
+* Show links to research, which statements or quotes are you hoping to use?
+  
+* Who might be a good person to interview for this piece?
+
+
+
```

Comment:
I've just finished reading this piece and I have a few thoughts and questions. Generally I think the golf use case for blockchain technology is quite interesting. But I think if we are going to report on this we need to make sure we cover the cypherpunk principles and ask unbiased questions. We also need to Digg deeper into the facts and produce evidence to backup our statements. Mainstream crypto media will always point in the direction of the positive but it's our job to scrutinise these views with counter arguments and evidence.

Firstly, it would have been good if you had put up an outline and thesis statement for this piece before writing. The reason for this is I'd like to start building our pieces around critical thinking and unbiased arguments. Asking basic questions like:

* Who does this new technology benefit? Who is aiming to profit?
* Does this new technology remove the middleman or just replace them?
* Is this new technology an improvement for the user or does it just digitise an already perfect system?
* Are these new systems going to be non-custodial and peer to peer or will they be controlled? If so, how?

As a final note to these questions, they should also be backed up with evidence e.g. This new system works in the following way ... which demonstrated that it is .... and here's my source.

From what I've read here it looks like the current systems of merchandise, handicaps grading, and scoring is being replaced by new centralised entities. In my opinion, this is decentralised in name only, with an aim to replace the current establishment with another. Do you see this differently? If so, please elaborate.

Do you think the systems being replaced are an improvement on the old ones? Do you see any potential issues with these new systems? If so, what issues do you see?

Reply:
I've addressed some points in the advantages and disadvantages but I will also add more detail